draft-mohan-asdf-sdf-protocol-mapping.md

```diff
@@ -139,6 +151,8 @@ where-
    protocol.
  - "zigbee" is an object containing properties that are specific to the
    Zigbee protocol.
+ - "http" is an object containing properties that are specific to the HTTP
+   protocol.
```

Comment:
As "HTTP" already includes "protocol", this could probably be shortened.

```suggestion
 - "http" is an object containing properties that are specific to HTTP.
```